web/src/types/tiktok-downloader.model.ts

```diff
@@ -0,0 +1,59 @@
+/** Interface for TikTokMedia */
+export interface TikTokMedia {
+    /** The format id of the media */
+    format_id: string 
+
+    /** The format note of the media */
+    format_note: string
+
+    /** The vcodec of the media */
+    vcodec: string
+
+    /** The url of the media */
+    url: string
+
+    /** The resolution of the media */
+    resolution: string
+}
+
+/** Interface for TikTokMeta */
+export interface TikTokMeta {
+    /** The name of the creator */
+    creator: string
+
+    /** The name of the uploader */
+    uploader: string
+
+    /** The title of the video */
+    title: string
+
+    /** The description of the video */
+    description: string
+
+    /** The duration of the video in string format */
+    duration_string: string
```

Comment:
please change this as discussed yesterday

Reply:
I wanted to create another Pull request to change everything regarding casing separately.